src/navigators/StackNavigator.js

```diff
@@ -42,7 +41,6 @@ export default (routeConfigMap: NavigationRouteConfigMap, stackConfig: StackNavi
   return createNavigationContainer(createNavigator(router)(props => (
     <CardStack
       {...props}
-      headerComponent={headerComponent}
```

Comment:
Shall we? It's beta and breaking changes are expected. Not sure that was a broadly used feature enough to deserve a special treatment in the codebase. I believe people at this point read release notes anyway.